plugins/modules/fusion_info.py

```diff
@@ -979,10 +981,13 @@ def generate_volumes_dict(module, fusion):
                     "storage_class": volume.storage_class.name,
                     "serial_number": volume.serial_number,
                     "target": {},
+                    "array": None,
                 }
+
                 # can be None if we don't have permission to see this
                 if volume.array is not None:
                     volume_info[vol_name]["array"] = volume.array.name
+
```

Comment:
Let's be consistent with the rest of the function (to improve readability of the code).
```suggestion
                    "array": getattr(volume.array, "name", None),
                }
```